### PR TITLE
Allowing number of replicas to be configurable

### PR DESCRIPTION
**Pull request by geethub97**
Resolves: https://github.com/rancher/helm-project-operator/issues/42

adding the ability to configure replicas in Helm Project Operator 

Related PR: https://github.com/rancher/helm-project-operator/pull/53

### CODE REVIEW

**Comment by aiyengar2** [comment]
Probably need to bump the Drone.yml to `rancher/dapper:v0.6.0`

File: .drone.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ steps:
     checksum:
     - sha256
     checksum_file: CHECKSUMsum-amd64.txt
-    checksum_flatten: true
+    checksum_flattzen: true
```
Comment (aiyengar2) [inline .drone.yml:27]: why did we modify this?

File: charts/helm-locker/templates/deployment.yaml [hunk 2]
```diff
@@ -6,7 +6,9 @@ metadata:
   labels: {{ include "helm-locker.labels" . | nindent 4 }}
     app: {{ template "helm-locker.name" . }}
 spec:
-  replicas: 1
+  {{- if .Values.replicas }}
```
Comment (aiyengar2) [inline charts/helm-locker/templates/deployment.yaml:9]: bump the chart version

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

File: .drone.yml (hunk 1 above)
Comment (geethub97) [reply .drone.yml:27]: this was modified when I ran go generate, but I will double check this again

**Review by aiyengar2 — APPROVED** [review]

**Pull request merged**